python/cudf/cudf/core/series.py

```diff
@@ -2788,6 +2788,19 @@ def cov(self, other, min_periods=None):
 
         return lhs._column.cov(rhs._column)
 
+    def transpose(self):
+        """Return the transpose, which is by definition self.
+        """
+
+        return self
+
+    @property
+    def T(self):
+        """Return the transpose, which is by definition self.
+        """
+
+        return self.transpose()
```

Comment:
```suggestion
    T = property(transpose,
        doc="Return the transpose, which is by definition self")
```

[Reference from pandas.](https://github.com/pandas-dev/pandas/blob/945c9ed766a61c7d2c0a7cbb251b6edebf9cb7d5/pandas/core/base.py#L290-L295)

Can you make this change to `IndexedFrame` and remove the counterpart in dataframe please?

Reply:
Thanks @isVoid for the reviews, for this particular comment, does it mean it is fine where the code is right now? Thanks for clarifying though, I was a bit unsure as to why would we remove `transpose` from `dataframe.py`

Reply:
Yeah it's fine to apply the above comment as is. You also shouldn't remove `transpose` in `dataframe.py`, but just apply the same changes above there.